### ISSUE TITLE
Option to increase default 40 char limit

### ISSUE THREAD

**Issue by OdayMansour** [kitplummer/clikan#51]
Would be nice to have a way to increase the 40 char limit on task names. 
French tends to have longer words so the limit becomes constraining.

(Willing to add this myself but wasn't sure if contributions to this project are welcome and how they should be done)

Loving this tool!

**Comment by kitplummer** [comment]
Of course!!  If it is cool to make the default length a configuration item
in local config, that would be awesome!

Thanks for the feedback too.  If you submit a PR I’ll look at it as soon as
I can.  It it is possible to add a functional test too, that verifies the
config setting all the better.

**Comment by OdayMansour** [comment]
Sounds good, PR submitted.

**Comment by kitplummer** [comment]
I’ll take a look tonight.  Thanks!
